README.md

```diff
@@ -489,7 +488,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ayushi Vamne](https://github.com/Ayushi15-Vamne) 
   - [Aziz Prabowo](https://github.com/azizp128)
   
-| [Back To Top](#contents) |
+| [`Back To Top`](#contents) |
```

Comment:
I don't think code backticks are needed for this (same for all the other instances). Not sure what others think, though.

Reply:
Ok..and thanks for replacement of those mistakes.